src/views/drivingLicence/check-your-details.html

```diff
@@ -80,31 +80,87 @@ <h1 id="header" data-page="{{hmpoPageKey}}" class="govuk-heading-l">
                     value: {
                         text: values.drivingLicenceNumber
                     }
+                },
+                {
+                    key: {
+                        text: translate("pages.check-your-details.issueNumber"),
+                        classes: "govuk-!-font-weight-bold"
+                    },
+                    value: {
+                        text: values.issueNumber
+                    }
+                },
+                {
+                    key: {
+                        text: translate("pages.check-your-details.postcode"),
+                        classes: "govuk-!-font-weight-bold"
+                    },
+                    value: {
+                        text: values.postcode
+                    }
                 }
             ]
         }) }}
+        {% endif %}
 
-        {% if values.licenceIssuer == "DVLA" %}
+        {% if values.licenceIssuer == "DVA" %}
 
             {{ govukSummaryList({
-                classes: "govuk-!-margin-bottom-0",
```

Comment:
Is removing this class intentional?

Reply:
Yes, this was originally in so that we could make two summary lists look like one (to reduce duplicated code we had a separate summary list to cover DVA/DVLA specific values) for this to look like one list we had to remove the spacing from the initial summary list. 

Now that we are making 1 summary list per issuer rather than a summary list for issuer specific fields, we have removed this css class so that the proper spacing is applied to the bottom of the summary list